desktop-displaymanagers/greetd-regreet/autobuild/defines

```diff
@@ -0,0 +1,13 @@
+PKGNAME=greetd-regreet
+PKGSEC=x11
+PKGDEP="gcc-runtime glib glibc greetd gtk-4 pango"
+BUILDDEP="rustc llvm"
+PKGSUG="sway"
```

Comment:
Just out of curiosity, what does sway do for this package?

Reply:
regreet need a Wayland compositor to launch